File: app/src/main/java/eu/toma/dev/playground/warcraft/model/item/BonusSummary.java
package eu.toma.dev.playground.warcraft.model.item;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class BonusSummary {

    @SerializedName("defaultBonusLists")
    @Expose
    public List<Object> defaultBonusLists = null;
    @SerializedName("chanceBonusLists")
    @Expose
    public List<Object> chanceBonusLists = null;
    @SerializedName("bonusChances")
    @Expose
    public List<Object> bonusChances = null;
}

File: app/src/main/kotlin/warcraft/model/item/BonusSummary.kt
package warcraft.model.item

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class BonusSummary {

    @SerializedName("defaultBonusLists")
    @Expose
    var defaultBonusLists: List<Any>? = null
    @SerializedName("chanceBonusLists")
    @Expose
    var chanceBonusLists: List<Any>? = null
    @SerializedName("bonusChances")
    @Expose
    var bonusChances: List<Any>? = null
}